### ISSUE TITLE
[Bug]: Xiaomi Home 自定义集成尝试与小米设备的 mips 服务重连多次但未成功

### ISSUE THREAD

**Issue by ertyy998** [XiaoMi/ha_xiaomi_home#208]
### Describe the bug / 描述问题

与小米设备的 mips 服务重连多次但未成功

### To Reproduce / 复现步骤

![屏幕截图 2024-12-19 102958](https://github.com/user-attachments/assets/5ace9c28-e48c-4223-9980-b8c5fee8ad0e)

### Expected behavior / 预期结果

反复出现了多次

### Home Assistant Logs / 系统日志

[home-assistant_2024-12-19T02-31-28.635Z.log](https://github.com/user-attachments/files/18191520/home-assistant_2024-12-19T02-31-28.635Z.log)

### Home Assistant Core version / Home Assistant Core 版本

2024.12.4

### Home Assistant Operation System version / Home Assistant Operation System 版本

13.2.1

### Xiaomi Home integration version / 米家集成版本

v0.1.2

**Comment by zghnwsq** [comment]
参考 #139

**Comment by Asssilem** [comment]
It seems that Xiaomi products are very convenient but not very stable. My Xiaomi robot vacuum cleaner often has app errors. [Unblocked Games 76](https://unblockedgames-76.io)

**Comment by zghnwsq** [comment]
> It seems that Xiaomi products are very convenient but not very stable. My Xiaomi robot vacuum cleaner often has app errors. [Unblocked Games 76](https://unblockedgames-76.io)

What‘s the link for? AD?

**Comment by zghnwsq** [comment]
> It seems that Xiaomi products are very convenient but not very stable. My Xiaomi robot vacuum cleaner often has app errors. [Unblocked Games 76](https://unblockedgames-76.io)

What‘s the link for? AD?

**Comment by topsworld** [comment]
Duplicate of #139

**Comment by topsworld** [comment]
非问题，等待中枢版本。
